charts/plgd-hub/values.yaml

```diff
@@ -27,6 +27,8 @@ global:
 #       useInUi: true
    web:
     clientID:
+  # -- Custom CA certificate for authorization endpoint in PEM format
+  authorizationCaPool:
```

Comment:
Please update also doc repository, and when this is merged, pretty the same MR will be needed in the dps repository. Copypaste.

Reply:
Would you like to add new section regarding custom ca config here ? https://github.com/plgd-dev/doc/blob/main/content/en/docs/deployment/k8s.md

Reply:
It would be very nice :-)
Text like - if you have external authorization, signed by not trusted CA, you can use this property…

Reply:
You can find MR here https://github.com/plgd-dev/doc/pull/81